### ISSUE TITLE
Table useful demand households displays a wrong total due to aggregate values shown

### ISSUE THREAD

**Issue by Charlottevm** [quintel/etmodel#4210]
![afbeelding](https://github.com/quintel/etmodel/assets/74194356/7ab75087-0679-4bdb-ad8d-c9ae9f6c7af9)

**Comment by noracato** [comment]
Good catch! I can quickly remove the totals column for this table. Or I could take a bit more time and exclude the other total rows from the super total, but I'm not sure if we can make that in time for the next deploy.

**Comment by noracato** [comment]
I went for the first option, because I could do it quickly. I we prefer the second option, please reopen the issue and I'll put it on the backlog
